src/redux/books/books.js

```diff
@@ -1,24 +1,43 @@
 const ADD_BOOK = 'bookstore/Books/ADD_BOOK';
 const REMOVE_BOOK = 'bookstore/Books/REMOVE_BOOK';
 
-const initialState = {
-  book: [],
-};
+const initialState = [
+  {
+    title: 'Economics new',
+    author: 'Elen',
+    id: 1,
+  },
+  {
+    title: 'The book',
+    author: 'Thomas Jeferson',
+    id: 2,
+  },
+  {
+    title: 'The world',
+    author: 'Jone Thomas',
+    id: 3,
+  },
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you assign your `initialState` to an empty array as now the user is able to ADD and REMOVE new books.

Reply:
Oka thanks I will do in now.